packages/apps/admin/src/components/Navigation/menuType/ActionEntry.js

```diff
@@ -1,23 +1,49 @@
 import PropTypes from 'prop-types'
+import {actions} from 'tocco-app-extensions'
 import {Icon} from 'tocco-ui'
 
-import {StyledIconLink, StyledMenuLink, StyledMenuLinkWrapper} from './StyledComponents'
+import {StyledIconLink, StyledMenuLink, StyledMenuLinkWrapper, StyledMenuText} from './StyledComponents'
 
-const ActionEntry = ({onClick, item}) => (
-  <StyledMenuLinkWrapper>
-    <StyledMenuLink data-quick-navigation={true} onClick={onClick} to={`/e/action/${item.name}`}>
-      {item.label}
-    </StyledMenuLink>
-    <StyledIconLink to={`/e/action/${item.name}`} target="_blank" rel="noreferrer">
-      <Icon icon="external-link" />
-    </StyledIconLink>
-  </StyledMenuLinkWrapper>
-)
+const ActionEntry = ({onClick, item}) =>
```

Comment:
suggestion(non-blocking): I personally prefer an if-statement to check for correct data

```
const ActionEntry = ({onClick, item}) => {
  if (!isCustomAction(item)) {
    return null // or return <></>
  }

  return (
    <StyledMenuLinkWrapper>
      {item.fullscreen ? (<>...</>) : (<StyledMenuText>...</StyledMenuText>)}
    </StyledMenuLinkWrapper>
  )
}
```